src/borg/archiver.py

```diff
@@ -1322,6 +1322,8 @@ def build_parser(self, prog=None):
                                   help='set umask to M (local and remote, default: %(default)04o)')
         common_group.add_argument('--remote-path', dest='remote_path', metavar='PATH',
                                   help='set remote path to executable (default: "borg")')
+        common_group.add_argument('--remote-ratelimit', dest='remote_ratelimit', type=int, metavar='rate',
+                                  help='set remote network rate limit in kbyte/s (default: unlimited)')
```

Comment:
also, you only limit local -> remote traffic, so it is useful for borg create, but not borg extract.